misc/workflow-environment-basic-dependencies-without-workspace/README.md

```diff
@@ -0,0 +1,33 @@
+## What is this?
```

Comment:
I think the name `workflow-environment-basic-dependencies-without-workspace` is explaining a problem you "used to have" (it won't exist here anymore) - so maybe you should give a more descriptive name like what is this actually used for

Reply:
@avnerenv0 but this is a public templates repo, so I didn't think I need to talk about integration tests here. Maybe i'll rename the other to `-with-workspace` ? WDYT?

Reply:
I think those are small "implementation" details that might change over time and are not capturing the core purpose of the template.
I'd rather name them "1" and "2"

Reply:
BUT really this is minor feel free to keep it like this and move on